x-pack/plugins/cases/public/components/user_action_tree/helpers.tsx

```diff
@@ -23,10 +23,11 @@ import {
   CommentType,
   Comment,
   CommentRequestActionsType,
+  noneConnectorId,
 } from '../../../common';
 import { CaseUserActions } from '../../containers/types';
 import { CaseServices } from '../../containers/use_get_case_user_actions';
-import { parseString } from '../../containers/utils';
+import { parseStringAsConnector, parseStringAsExternalService } from '../../containers/utils';
```

Comment:
I think if utility functions are used both by the containers and the components it is best to move them on an upper level like `x-pack/plugins/cases/public/common/utils.ts`.